app/pages/hosting/softaculous.html

```diff
@@ -0,0 +1,195 @@
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-database" style="color: #e67e22"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Backups / Restore</h3>
+            <p>Configure backups which can be restored if anything goes wrong</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-download" style="color: #2ecc71"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Clone Installations</h3>
+            <p>Clone your website to test any changes before you make it to your live site</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-wordpress" style="color: #0087be"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Wordpress Themes</h3>
+            <p>Install WordPress themes while installing WordPress itself</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-globe" style="color: mediumslateblue"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Multilingual</h3>
+            <p>Most of the apps can be installed if your native language</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-globe" style="color: mediumslateblue"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>No Domain? No Worries</h3>
+            <p>
+              Don't have a domain yet? No problem, get a Life Time FREE Kloud51 Sub Domain. You can change it to a
+              domain later if you like to. We have your back to run your website with or without domain.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-6 col-lg-offset-3 grid">
+        <div class="row">
+          <div class="col-sm-3">
+            <i class="fa fa-search" style="color: #2ecc71"></i>
+          </div>
+          <div class="col-sm-9">
+            <h3>Malware Scan &amp; Removal</h3>
+            <p>
+              Instant &amp; Real-time Malware/Virus detection. While you're rocking with your website, we're protecting
+              you from any malicious activities and files. Relax, just keep up the awesome work.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="text-center s-pad-top">
+      <a href="https://kloud51.com/cart.php?gid=1" class="btn btn-success btn-launch">
+        <i class="fa fa-rocket fa-fw"></i>  Launch Your Site NOW!        
```

Comment:
One space is enough.

Reply:
Done.

Reply:
Actually both btn-launches had 1 unwanted spaces.